doc/nrf/libraries/caf/buttons_pm_keep_alive.rst

```diff
@@ -0,0 +1,26 @@
+.. _caf_buttons_pm_keep_alive:
+
+CAF: Power manager keep alive module for buttons
+################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |buttons_pm_keep_alive| of the :ref:`lib_caf` (CAF) is responsible for generating :c:struct:`keep_alive_event` event on any configured button press.
+
+Configuration
+*************
+
+To use the module, you must enable the following Kconfig options:
+
+* :kconfig:`CAF_BUTTONS_PM_KEEP_ALIVE` - This option enables the |buttons_pm_keep_alive|.
+* :kconfig:`CONFIG_CAF_BUTTONS` - This option enables the buttons module.
+* :kconfig:`CAF_KEEP_ALIVE_EVENTS` - This option enables keep alive event.
```

Comment:
This is automatically enabled now, no need to enable it.

Reply:
Replaced by:
```
* :kconfig:`CAF_POWER_MANAGER` - This option enables power manager and is required by :kconfig:`CAF_BUTTONS_PM_KEEP_ALIVE`.
```